app/components/QrCodeScanner/QrCodeScanner.jsx

```diff
@@ -29,24 +51,99 @@ export default class QrCodeScanner extends Component<Props> {
       callback(content, this.stopScanner.bind(this))
     })
 
-    Instascan.Camera.getCameras()
+    // since the browser halts while getting usermedia
+    // let the loading animation finish one round and only then continue
+    this.setState({ loading: true })
+    new Promise(resolve => setTimeout(resolve, 900))
```

Comment:
Is there a reason we are using 900, maybe put in a variable with a name that is more explanatory.